index.bs

```diff
@@ -3498,8 +3498,8 @@ When registering a new credential, represented by an {{AuthenticatorAttestationR
         - If [=self attestation=] was used, verify that [=self attestation=] is acceptable under [=[RP]=] policy.
         - If [=ECDAA=] was used, verify that the [=identifier of the ECDAA-Issuer public key=] used is included in the set of
```

Comment:
I believe the same change should be applied for ECDAA as well. [6.4.2. Attestation Statement Formats](https://w3c.github.io/webauthn/#attestation-formats) states:

> This attestation trust path is either empty (in case of self attestation), an identifier of an ECDAA-Issuer public key (in the case of ECDAA), or a set of X.509 certificates.

Rest looks LGTM!

Reply:
Good point, thanks!